ground_truth_labeling_jobs/ground_truth_conversion_scripts/GT_semantic_segmentation_to_COCO.ipynb

```diff
@@ -0,0 +1,443 @@
+    "    \"categories\": categories,\n",
+    "    \"images\": images,\n",
+    "    \"annotations\": annotations\n",
+    "    \n",
+    "}\n",
+    "with open('COCO_file.json', 'w') as json_file:\n",
+    "        json.dump(COCO_json, json_file)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Test and Visualize Each Unique Category Segmentation\n",
+    "\n",
+    "This section is based on this notebook https://github.com/cocodataset/cocoapi/blob/master/PythonAPI/pycocoDemo.ipynb"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "from pycocotools.coco import COCO\n",
+    "import numpy as np\n",
+    "import skimage.io as io\n",
+    "import matplotlib.pyplot as plt\n",
+    "import pylab\n",
+    "pylab.rcParams['figure.figsize'] = (20.0, 10.0)\n",
+    "\n",
+    "annFile='COCO_file.json'\n",
+    "coco=COCO(annFile)\n",
+    "# display COCO categories and supercategories\n",
+    "cats = coco.loadCats(coco.getCatIds())\n",
+    "nms=[cat['name'] for cat in cats]\n",
+    "print('COCO categories: \\n{}\\n'.format(' '.join(nms)))\n",
+    "\n",
+    "nms = set([cat['supercategory'] for cat in cats])\n",
+    "print('COCO supercategories: \\n{}'.format(' '.join(nms)))\n",
+    "\n",
+    "\n",
+    "catIds = coco.getCatIds(catNms=['bike','motorbike']);\n",
+    "imgIds = coco.getImgIds(catIds=catIds );\n",
+    "imgIds = coco.getImgIds(imgIds = [1]) #<<<<<<<<<<<< Change the number for different test images\n",
+    "img = coco.loadImgs(imgIds[np.random.randint(0,len(imgIds))])[0]\n",
+    "print(img)\n",
+    "\n",
+    "I = io.imread(img['file_name'])\n",
+    "h, w, c = I.shape\n",
+    "print(h)\n",
+    "print(w)\n",
+    "plt.axis('off')\n",
+    "plt.imshow(I)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Print out the annotations"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.imshow(I); plt.axis('off')\n",
+    "annIds = coco.getAnnIds(imgIds=img['id'], catIds=catIds, iscrowd=None)\n",
+    "anns = coco.loadAnns(annIds)\n",
+    "coco.showAnns(anns)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Print and show the segmentation using COCO format"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pycocotools.coco as coco\n",
+    "from pycocotools.coco import COCO\n",
+    "import cv2\n",
+    "\n",
+    "for annotation in annotations:\n",
+    "    with open('COCO_file.json', 'w') as json_file:\n",
+    "        json.dump(annotation, json_file)\n",
+    "    coco=COCO('COCO_file.json')\n",
+    "    I = io.imread('<sample_image>')\n",
+    "    plt.imshow(I)\n",
+    "    coco.showAnns([annotation])"
```

Comment:
Is this supposed to be loading all of the images and plotting the annotations? If so, you might need to make a new figure for each image (or maybe plot out to a pdf), and add logic to get the image ID from the annotation.

Reply:
I changed it to just plot one random image but ploting all images in a pdf sounds like a great idea so I can do that as an improvement